modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -186,22 +186,36 @@ class ApiClient(object):
             # use server/host defined in path or operation instead
             url = _host + resource_path
 
-        # perform request and return response
-        response_data = {{#asyncio}}await {{/asyncio}}{{#tornado}}yield {{/tornado}}self.request(
-            method, url, query_params=query_params, headers=header_params,
-            post_params=post_params, body=body,
-            _preload_content=_preload_content,
-            _request_timeout=_request_timeout)
-
-        self.last_response = response_data
-
-        return_data = response_data
-        if _preload_content:
-            # deserialize response data
-            if response_type:
-                return_data = self.deserialize(response_data, response_type)
-            else:
-                return_data = None
+        try:
+            # perform request and return response
+            response_data = {{#asyncio}}await {{/asyncio}}{{#tornado}}yield {{/tornado}}self.request(
+                method, url, query_params=query_params, headers=header_params,
+                post_params=post_params, body=body,
+                _preload_content=_preload_content,
+                _request_timeout=_request_timeout)
+
+            content_type = response_data.getheader('content-type')
+
+            self.last_response = response_data
+
+            return_data = response_data
+            if _preload_content:
+                if six.PY3:
+                    if response_type not in ["file", "bytes"]:
```

Comment:
Should we do this for file-type only now?

Reply:
@spacether If possible, I'd like to leave the groundwork for the next PR if at all possible (if it's not hurting anything).

Reply:
As long as we remove the bytes addition from the python generator in the java files then I am fine with that.
That would make it so we would never see this "bytes" type until your later PR.